oc_dms_mirror/dms_mirror.py

```
@@ -190,6 +190,11 @@ def register_component(self, payload):
 
         # Component only deliverable if the labels non-deliverable not in component labels
         is_deliverable = False if "non-deliverable" in component_labels else True
+        should_registered = False if "test-component" in component_labels else True
+
+        if not should_registered:
+            self.logger.info(f"Component with id [{component_id}] should not be registered, skipping")
```

Ruff (0.15.5)

[warning] Logging statement uses f-string

(G004)